### DISCUSSION TITLE
How to Integrate  React Epub Reader with next js

### DISCUSSION THREAD

**Discussion by sadam1807** [vercel/next.js#20568] [Help]
Hello 
I am trying to integrate [react epub reader ](https://www.npmjs.com/package/react-reader) with Next but getting one error but works with react,  **I need your help to integrate epub reader with nextjs**

![image](https://user-images.githubusercontent.com/42937204/103280841-64185c80-49f7-11eb-8ec8-5d240f7ced51.png)

 
this is my code

> import React from 'react'
import { ReactReader } from "react-reader";

export default function Home() {
  return (
    <div>
         <div style={{ position: "relative", height: "330px" }}>
          
          <ReactReader
            url="./Boy without a name English.epub"
            title={"Alice in wonderland"}
            location={"epubcfi(/6/2[cover]!/6)"}
            locationChanged={epubcifi => console.log(epubcifi)}
          />
        </div>
      </div>
  )
}

**Comment by lachlanjc** [comment]
This question was answered in the issue: https://github.com/vercel/next.js/issues/20569#issuecomment-752864145
